i18n/de.json

```diff
@@ -5,14 +5,14 @@
   "app.about.version": "Version: {version}",
   "app.aura.updatedisabled": "Updates deaktiviert",
   "app.aura.updateready": "Update verfügbar",
-  "app.aura.versionskip": "version skipped",
+  "app.aura.versionskip": "Version übersprungen",
   "app.config.account.notvalidtooltip": "Wir konnten keine WeakAura Dateien auf diesem Account finden. Hast du das AddOn installiert?",
   "app.config.account.selectaccounttooltip": "Wähle einen Account",
   "app.config.autoStart": "Client starten, wenn mein Computer gestartet wird",
   "app.config.autoupdater": "Auto-Update",
-  "app.config.autoupdater.autoupdate": "Install Companion updates automatically",
-  "app.config.autoupdater.beta": "Use Companion Beta channel",
-  "app.config.autoupdater.check": "Check for COmpanion Updates",
+  "app.config.autoupdater.autoupdate": "Installiere Companion Updates Automatisch",
```

Comment:
*updates automatisch

Reply:
"automatisch" is now lowercase while "(Die) Updates" is a noun